cdk/csharp/README.md

```diff
@@ -0,0 +1,28 @@
+# How to deploy the AWS Lambda Power Tuner tool using the CDK for C#
+
+This CDK project deploys the *AWS Lambda Power Tuner* tool.
+
+You can use the project as a standalone or reuse it within your own CDK projects.
+
+## Prerequisites
+
+- [AWS CDK Toolkit](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_install)
+- [.NET 8.0 or later](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_prerequisites)
```

Comment:
I specified .NET 8.0 here, also used in projects (CDK guide says .NET 6.0)

Reply:
@andrestoll @Anirudh1612 I'm wondering if we should standardize on one single README for all the CDK sub-folders? 

See the TypeScript PR here too: https://github.com/alexcasalboni/aws-lambda-power-tuning/pull/230

If it does make sense to keep individual README files for each language, how do we make each one useful with language-specific tips or guidance?